crates/ruff_linter/src/rules/pydocstyle/rules/triple_quotes.rs

```diff
@@ -78,12 +78,14 @@ pub(crate) fn triple_quotes(checker: &mut Checker, docstring: &Docstring) {
                 let mut diagnostic =
                     Diagnostic::new(TripleSingleQuotes { expected_quote }, docstring.range());
 
-                let body = docstring.body().as_str();
-                if !body.ends_with('\'') {
-                    diagnostic.set_fix(Fix::safe_edit(Edit::range_replacement(
-                        format!("{prefixes}'''{body}'''"),
-                        docstring.range(),
-                    )));
+                if checker.settings.preview.is_enabled() {
```

Comment:
There’s a risk that this is confusing since the rule will be listed as fixable in the docs, but it is consistent with the versioning policy.

Reply:
That's a good point, I think we'll need to address that separately from this release though unless you want to delay.

Reply:
#8075